config/ufs/machines/config_machines.xml

```diff
@@ -299,6 +299,64 @@ This allows using a different mpirun command to launch unit tests
     </resource_limits>
   </machine>
 
+
+  <machine MACH="hera">
+    <DESC>NOAA hera system</DESC>
+    <NODENAME_REGEX>hfe</NODENAME_REGEX>
+    <OS>LINUX</OS>
+    <COMPILERS>intel</COMPILERS>
+    <MPILIBS>impi</MPILIBS>
+    <PROJECT>nems</PROJECT>
+    <SAVE_TIMING_DIR/>
+    <CIME_OUTPUT_ROOT>/scratch1/NCEPDEV/nems/$USER</CIME_OUTPUT_ROOT>
+    <DIN_LOC_ROOT>/scratch1/NCEPDEV/nems/Rocky.Dunlap/INPUTDATA</DIN_LOC_ROOT>
+    <DIN_LOC_ROOT_CLMFORC>/scratch1/NCEPDEV/nems/Rocky.Dunlap/INPUTDATA/atm/datm7</DIN_LOC_ROOT_CLMFORC>
+    <DOUT_S_ROOT>$CIME_OUTPUT_ROOT/archive/$CASE</DOUT_S_ROOT>
+    <BASELINE_ROOT>/scratch1/NCEPDEV/nems/Rocky.Dunlap/BASELINES</BASELINE_ROOT>
+    <CCSM_CPRNC>/home/Rocky.Dunlap/bin/cprnc</CCSM_CPRNC>
```

Comment:
This will need to be changed.

Reply:
I tested out your changes and got through the build stage, but the chgres run failed because the template.chgres.run file specifies "-np" in the $mpirun command on line 56. Since the "mpirun" command on hera is actually "srun", the flag needs to be just "-n". Once I changed the template, both chgres and ufs ran to completion. Unfortunately, I don't know offhand if we can just change that to -n for all versions of MPI or not.